library/build.gradle

```diff
@@ -12,10 +12,10 @@ sourceCompatibility = 1.8
 
 repositories {
     mavenCentral()
-    jcenter()
 }
 
 jar {
```

Comment:
Actually... would it be possible to (also?) publish this library as a **thin jar** (as opposed to fat jar)? i.e. without all the dependencies, like kotlin-stdlib (or log4j, mentioned separately below) baked in? 🤔

Reply:
Ok! subscribed 👁️